timeless/poster/tasks.py

```diff
@@ -46,11 +46,11 @@ def sync_tables():
 def sync_customers():
     """
     Periodic task for fetching and saving tables from Poster
+    Docs - https://dev.joinposter.com/docs/api#clients-getclients
     """
     for poster_customer in __poster_api().customers():
         customer = DB.session(Customer).query.filter_by(
-            first_name=poster_customer["first_name"],
-            last_name=poster_customer["last_name"]
+            poster_id=poster_customer["client_id"],
```

Comment:
@vladarefiev I don't think this change is ok here, right now. I am also reviewing [this](https://github.com/timelesslounge/timelessis/pull/522) PR (solution for #410) and the logic of the merge method is based on the fact that customers are identified by first and last name here.

Reply:
@amihaiemil Thanks, done